index.html

```diff
@@ -0,0 +1,490 @@
+                <div class="col-8 left-column">
+                    <section>
+                        <div class="video-container">
+                            <iframe src="https://www.youtube.com/embed/V74l_zS1x8E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                        </div>
+                        <div class="hashtags">
+                            #forextrading #forex #trading 
+                        </div>
+                        <div class="video-title">
+                            <h5>Contributing to Open Source Part I: The Easy Way</h5> 
+                        </div>
+                        <div class="video-description">
+                            <p class="desc-text"><span class="views-and-date">67,688 views  Feb 27, 2014</span>
+                                How do you contribute to open source?  In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github.</p>
+                        </div>
+                        <div>
+                            <ul class="navbar-nav flex-row flex-wrap ms-md-auto">
+                                <li class="nav-item">
+                                    <i data-feather="thumbs-up" class="mb-1"></i> 46K
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="thumbs-down" class="mb-1"></i> Dislike
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="share-2" class="mb-1"></i> Share
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="scissors" class="mb-1"></i> Clip
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="plus-square" class="mb-1"></i> Save
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="creator-details mt-3">
+                            <div class="container">
+                                <div class="row justify-content-between">
+                                    <div class="d-flex justify-content-between col-md-12 col-sm-12 col-lg-12 col-xl-5 col-xxl-5 border border-secondary one-of-two-box p-1">
+                                        <div class="creator-left col-6 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in the external CSS file you created.